packages/account/src/Components/self-exclusion/self-exclusion-inputs.tsx

```diff
@@ -3,11 +3,36 @@ import classNames from 'classnames';
 import { Button, DatePicker, Icon, Input, Text } from '@deriv/components';
 import { getBrandWebsiteName, epochToMoment, toMoment, PlatformContext, isMobile } from '@deriv/shared';
 import { Localize, localize } from '@deriv/translations';
-import { Field, useFormikContext } from 'formik';
+import {
+    Field,
+    FormikComputedProps,
+    FormikErrors,
+    FormikHandlers,
+    FormikHelpers,
+    FormikState,
+    FormikValues,
+    useFormikContext,
+} from 'formik';
 import SelfExclusionContext from './self-exclusion-context';
 import SelfExclusionFooter from './self-exclusion-footer.jsx';
 
-const SectionTitle = ({ title, has_border_line }) => {
+type TSectionTitle = {
+    title: React.ReactElement;
+    has_border_line?: boolean;
+};
+
+type TFormikContext = {
+    errors: FormikErrors<FormikValues>;
+    handleBlur: FormikHandlers['handleBlur'];
```

Comment:
@niloofar-deriv I remember that you have handleBlur in one of your previous PRs. was it also from Formik library?
because I didn't know about this type before. maybe you can update your code like this as well . could you check please?

Reply:
@shayan-deriv thanks for mentioning this, it's different from my component, I used handleBlur in the div element